contrib/osx/make_osx.sh

```diff
@@ -154,6 +154,54 @@ PYINSTALLER_COMMIT="413cce49ff28d87fad4472f4953489226ec90c84"
 info "Installing PyInstaller."
 python3 -m pip install --no-build-isolation --no-dependencies --no-warn-script-location "$CACHEDIR/pyinstaller"
 
+
+info "DEBUG SATOCHIP START"
+
+info "Installing some build-time deps for compilation..."
+brew install autoconf automake libtool gettext coreutils pkgconfig
+
+
+info "Building PyInstaller."
```

Comment:
this whole block is duplicated from above?
see line 123

Reply:
Yes, sorry, the MacOS build is still a work in progress, I pushed the changes I made so far to the Satochip repo in order to work on the mac build from my mac...

Reply:
If this PR is still in progress better to change it into a draft